fleet-be/src/main/java/com/everis/training/fleet/business/fleet/entity/Reserve.java

```diff
@@ -1,15 +1,15 @@
 package com.everis.training.fleet.business.fleet.entity;
 
 public class Reserve {
```

Comment:
"Reservation" would be a more accurate name since "Reserve" is a verb and represents an action.